lib/danger/scm_source/git_repo.rb

```diff
@@ -19,6 +19,26 @@ def diff_for_folder(folder, from: "master", to: "HEAD")
       self.log = repo.log.between(from, to)
     end
 
+    def renamed_files
+      before_name_regexp = /rename from (.*)rename/
+      after_name_regexp = /rename to (.*)$/
+
+      self.diff.map do |diff_file|
+        patch = diff_file.patch.tr("\n", "")
+
+        before_match = patch.match(before_name_regexp)
+        next unless before_match
+
+        after_match = patch.match(after_name_regexp)
+        next unless after_match
+
+        {
+          before: before_match.captures.first,
+          after: after_match.captures.first
```

Comment:
No, with `--diff-filter=R`, diff will not contain file contents diff and will formated like this

```
diff --git a/hello1 b/hello1_renamed
similarity index 100%
rename from hello1
rename to hello1_renamed

diff --git a/hello3 b/hello3_renamed
similarity index 100%
rename from hello3
rename to hello3_renamed

diff --git a/hello5 b/hello5/some_folder/hello5
similarity index 100%
rename from hello5
rename to hello5/some_folder/hello5
```